packages/vscode-extension/src/project/project.ts

```diff
@@ -96,6 +97,11 @@ export class Project implements Disposable, MetroDelegate, ProjectInterface {
     this.workspaceWatcher.onDidChange(() => this.checkIfNativeChanged());
     this.workspaceWatcher.onDidCreate(() => this.checkIfNativeChanged());
     this.workspaceWatcher.onDidDelete(() => this.checkIfNativeChanged());
+    this.fileSaveWatcherDisposable = workspace.onDidSaveTextDocument(({ fileName }) => {
+      if (fileName.includes("node_modules")) {
```

Comment:
Is it necessary to do the filename check here? It seems like it should matter as otherwise we'd get the watcher triggered. If that's the case I'd just skip it given we have throttling in place and that it'd be more error prone to some other possible updates people are making

Reply:
I initially wanted to await running fingerprinting all the time when people edit their files but we already have fs watchers in place. Will change.